tools/runners/run-proxy.js

```diff
@@ -124,6 +125,7 @@ Object.assign(Proxy.prototype, {
             'Content-Type': 'text/plain'
           });
         }
+        console.error("Proxying error", err);
```

Comment:
All other logs here use `runLog.log(...)`. Are you sure we need this? In which scenario will this show?

Reply:
It was silently failing all proxying attempts so I couldn't debug my problem. I moved it so it's returned in the reply instead of logging now (runLog doesn't work for this path):
<img width="750" alt="image" src="https://github.com/user-attachments/assets/9fe23711-0144-4e99-b785-d8e190a0026a" />